<!--@file lane_detection_left_cam.launch
 -  @brief This launch file is used to run the lane detection node on the
 -       	 left camera
 -  @author Matthew Post
 -  @author Toni Ogunmade(oluwatoni)
 -  @competition IARRC 2018
 -->
<launch>
	<arg name="direction" default="left"/>
	<node pkg="rr_lane_detection" type="lane_detection" name="lane_detection_left" output="screen"/>
	<param name="Debug_Flag" type="bool" value="true" />
	
	<param name="Camera_Source_Topic" type="string" value="/rr_vehicle/left_cam/image_raw"/>
	<param name="Output_Image_topic" type="string" value="/output_video_left" />
	<param name="Output_Pointlist_topic" type ="string" value="/output_point_list_left"/>
	<param name="PointListOut" type="bool" value="true" />
	
	<param name="Blob_Size" type="int" value="8" />
	<param name="Fixed_Threshold" type="int" value="0" />
	<param name="Adaptive_HSV_Max_S" type ="int" value = "16"/>
	<param name="Adaptive_HSV_Min_V" type ="int" value = "-20"/>
	<param name="Adaptive_HSV_MPatch_Size" type ="int" value = "25"/>

	<param name="OpenCVMatFile" type="string" value="$(find rr_lane_detection)/config/vision_params_left.yaml" />
	<!-- FYI: OpenCV Image coordinates are in [x,y], where X is a column index (0 left), and Y is a row index (0 top) -->
</launch>
